### ISSUE TITLE
Projects app-cicd and infra-pipeline do not respect the guide naming convention

### ISSUE THREAD

**Issue by daniel-cit** [GoogleCloudPlatform/terraform-example-foundation-app#32]
## Expected Behavior

Projects `app-cicd` and `infra-pipeline` created in step [foundation-extension/4-projects](https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/blob/main/foundation-extension/4-projects) should respect the guide naming convention:

- prj-bu1-c-app-cicd
- prj-bu1-c-infra-pipeline

## Actual Behavior

Projects `app-cicd` and `infra-pipeline` do not respect the guide naming convention:

- prj-bu1-s-app-cicd
- prj-bu1-s-infra-pipeline

## Steps to Reproduce the Problem

1. Run the steps in [foundation-extension 4-projects README](https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/blob/main/foundation-extension/4-projects/README.md)
1. Check the project names in https://console.cloud.google.com/cloud-resource-manager

## Specifications

- Version: d92e89485d2b6ea1caf633b66b41d8a10380e64d
- Platform: linux

**Comment by daniel-cit** [comment]
The environment used, `shared`: 

https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/blob/6b79823dd64998b37cf58c218870d00a24a9b662/app-foundation/4-projects/shared/app_cicd.tf#L42

https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/blob/6b79823dd64998b37cf58c218870d00a24a9b662/app-foundation/4-projects/shared/infra_pipeline.tf#L23

should be `common` like it is used in the foundation

```terraform
module "app_infra_cloudbuild_project" {
  source                      = "../../modules/single_project"
  ...
  environment                 = "common"
```
https://github.com/terraform-google-modules/terraform-example-foundation/blob/e30fe8cbd267b9f2e5dad9bc8fdb4360880b1cd4/4-projects/business_unit_1/shared/example_infra_pipeline.tf#L23

It is also necessary to change the READMEs and tfvars files that have examples for this projects

**Comment by rutalreja-deloitte** [comment]
@daniel-cit can you link to the guide naming convention or suggest the appropriate names, I can make a PR
Comment 2 on `shared` vs `common` addressed in #36

**Comment by daniel-cit** [comment]
Hi @rutalreja-deloitte with the change you did in the PR https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/pull/36 the projects will follow the guide:

- `prj-bu1-s-app-cicd`         => `prj-bu1-c-app-cicd`
- `prj-bu1-s-infra-pipeline` => `prj-bu1-c-infra-pipeline`

These READMEs and tfvars also need the new names

- terraform-example-foundation-app/README.md
- terraform-example-foundation-app/5-infrastructure/README.md
- terraform-example-foundation-app/foundation-extension/README.md
- terraform-example-foundation-app/README.md
- terraform-example-foundation-app/5-infrastructure/README.md
- terraform-example-foundation-app/7-app-build-deploy/README.md
- terraform-example-foundation-app/foundation-extension/README.md
- terraform-example-foundation-app/5-infrastructure/business_unit_1/development/development.auto.example.tfvars
- terraform-example-foundation-app/5-infrastructure/business_unit_1/non-production/non-production.auto.example.tfvars
- terraform-example-foundation-app/5-infrastructure/business_unit_1/production/production.auto.example.tfvars
- terraform-example-foundation-app/5-infrastructure/business_unit_1/shared/shared.auto.example.tfvars
- terraform-example-foundation-app/foundation-extension/4-projects/business_unit_1/development/development.auto.example.tfvars
- terraform-example-foundation-app/foundation-extension/4-projects/business_unit_1/non-production/non-production.auto.example.tfvars
- terraform-example-foundation-app/foundation-extension/4-projects/business_unit_1/production/production.auto.example.tfvars

**Comment by rutalreja-deloitte** [comment]
@daniel-cit did a find and replace in the PR #36, let me know this addresses everything in the issue

**Comment by daniel-cit** [comment]
> @daniel-cit did a find and replace in the PR #36, let me know this addresses everything in the issue

LGTM

**Comment by bharathkkb** [comment]
fixed via #36
